src/main/java/no/difi/sdp/client2/domain/Noekkelpar.java

```diff
@@ -29,44 +30,60 @@
 public class Noekkelpar {
 
     private KeyStore keyStore;
-    private String alias;
-    private String password;
+    private KeyStore trustStore;
+    private String virksomhetssertifikatAlias;
+    private String virksomhetssertifikatPassord;
 
-    private Noekkelpar(KeyStore keyStore, String alias, String password) {
+    private static final String TRUST_STORE_PASSORD = "sophisticatedpassword";
```

Comment:
Bør disse ha litt tydeligere navngivning for å peke på hva de er? Er det for testkjøring? Er det noe lurendreieri som er bundla med biblioteket? Er det placeholders?